dadjokes-onchain/README.md

```diff
@@ -0,0 +1,15 @@
+# Dad Jokes On Chain
+> Dad jokes, they used to be off the chain.
+
+**_TODO: YT LINK_**
+This repo support the tutorial [Dad Jokes On Chain]()
+
+## How To Use The Repo
+The repo is broken into 2 main sections via folders
+**foundry**: This folder contains a foundry project with the smart contract which will hold the dad jokes and any reward that has been given to them. 
+
+**next**: This folder contains the next.js project which will act as the front end for interacting with the smart contract. 
```

Comment:
nit: should we adopt more standardized folder naming by calling it app or frontend rather than couple with the tool?